workspace/src/app/views/pages/Project/ActivityInfoWidget.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import {
+    Card,
+    IconButton,
+    Spacing,
+    OverviewItem,
+    OverviewItemDescription,
+    OverviewItemActions,
+    OverviewItemLine,
+} from "gui-elements";
+import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector } from "react-redux";
+import { commonSel } from "@ducks/common";
+import { routerOp } from "@ducks/router";
+import { SERVE_PATH } from "../../../constants/path";
+
+const ActivityInfoWidget = () => {
+    const projectId = useSelector(commonSel.currentProjectIdSelector);
+    const dispatch = useDispatch();
+    const [t] = useTranslation();
+
+    const projectPath = `projects/${projectId}/activities?page=1&limit=25`;
+    return (
+        <Card>
```

Comment:
Is there any reason why not to use the `Card` API to have a normal card header with options menu for the icon button here? At least pls create a follow up issue in JIRA for that widget.